### ISSUE TITLE
Could upgrade and support torch-geometric 2.0 ?

### ISSUE THREAD

**Issue by zsz00** [microsoft/Graphormer#77]
Could upgrade and support torch-geometric 2.0 ?

**Comment by shiyu1994** [comment]
@zsz00 Thanks for using Graphormer. I've tested before with torch-geometric 2.0.3. It works with Graphormer. So I think it is okay to install torch-geometric 2.0 and use it with Graphormer.

**Comment by zsz00** [comment]
yes, I tested with torch-geometric 2.0.3 , pytorch 1.10.1, cuda 11.3,  nvidia 3090, ubuntu 18.04 , is ok !!!

**Comment by YouJiacheng** [comment]
If you do not need oc20 dataset(which use torch-geometric 1.x and not compatible with 2.x), it's okay to use 2.0.

**Comment by skye95git** [comment]
> yes, I tested with torch-geometric 2.0.3 , pytorch 1.10.1, cuda 11.3, nvidia 3090, ubuntu 18.04 python3.9, is ok !!!

Why did you change the version? Is it because install.sh doesn't work properly?
